admin/bin/dev-shell

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -eu
+
+TOPLEVEL=$(git rev-parse --show-toplevel)
+
+function docker_image_circle() {
+    local out
+    out="$(mktemp)"
+    ( cat Dockerfile ; echo COPY . . ) > circle.docker
+    if ! docker build -t securedrop-admin -f circle.docker . >& "$out" ; then
+        cat "$out"
+        return 1
+    fi
+    return 0
+}
+
+function docker_run_circle() {
+    docker run --rm -ti securedrop-admin "$@"
+}
+
+function docker_image() {
+    local out
+    out="$(mktemp)"
+    if ! docker build \
+	 --build-arg=USER_ID="$(id -u)" \
+	 --build-arg=USER_NAME="${USER:-root}" \
+         -t securedrop-admin "$TOPLEVEL/admin" >& "$out" ; then
+        cat "$out"
```

Comment:
Minor dev UX issue: I get no feedback during the image build (can address simply by printing "Building image, this might take a few minutes..." before kicking off `docker build`)

Reply:
added something to display feedback, good point